components/Navbar/Tab.tsx

```diff
@@ -55,6 +55,7 @@ export default function Tab({
             'hover:bg-black/25 hover:text-white hover:text-opacity-100':
               status && status.unlocked && !isActive,
             'bg-black/25 text-opacity-100': isActive,
+            'border-r': isLast,
```

Comment:
Hey, @shaavan it seems like the little `isLast` variable may have caused a bit of trouble. It looks like it wasn't declared in the `components/Navbar/Tab.tsx` file. I'm surprised that Vercel wasn't able to catch this error, but I was able to spot it when I pulled the master branch locally.:
![Screenshot 2023-02-08 at 1 59 58 PM](https://user-images.githubusercontent.com/54861487/217476514-9d46509f-1294-47de-8d96-93cf934a4d06.png)

The issue with the `isLast` variable can also be seen in the commit history on GitHub, at https://github.com/saving-satoshi/saving-satoshi/commits/master. It seems that this pull request was unable to build due to the missing declaration.